locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -751,6 +751,10 @@ protected MachineLocation obtainOnce(ConfigBag setup) throws NoMachinesAvailable
                 machineCreationSemaphore.release();
             }
 
+            for (JcloudsLocationCustomizer customizer : customizers) {
+                customizer.customize(this, node, setup);
```

Comment:
Looks like this should be moved down slightly - we don't set `node` until just after this, so we'll always be passing `null` to the customizer.